multiapps-common/src/main/java/org/cloudfoundry/multiapps/common/util/XmlUtil.java

```diff
@@ -83,7 +83,7 @@ private XmlUtil() {
         return fromXml(xml, clazz, null);
     }
 
-    public static <T> T fromXml(Document xml, Class<T> clazz, URL schemaLocation) throws ParsingException {
+    private static <T> T fromXml(Document xml, Class<T> clazz, URL schemaLocation) throws ParsingException {
```

Comment:
why thou?

Reply:
Because `Document` comes from `javax.xml.bind` and if it remained public I would've had to use `requires transitive` in this module's `module-info.java` file.